selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -576,12 +576,6 @@
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewNotJavaFilesTest"/>
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewPackageFromContextMenuTest"/>
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewPackagesWithHelpCreationJavaClassTest"/>
-            <class name="org.eclipse.che.selenium.projectexplorer.CreatePackagesAndFilesWithHelpIconCreateTest">
```

Comment:
Can you elaborate on why some tests are removed from an xml configs and from java classes?

Reply:
we can do it if and when we will be testing Codenvy on SPI